2025-02-06 00:01:18,141 INFO Logger is configured with simple rotating file handler.
2025-02-06 00:01:18,141 INFO Assets path configured: C:\Users\K.T\Documents\Github\R_ScrapingRank\assets
2025-02-06 00:01:18,148 INFO Starting up TCP server on localhost:50762
2025-02-06 00:01:18,148 INFO Flet app has started...
2025-02-06 00:01:18,148 INFO App URL: tcp://localhost:50762
2025-02-06 00:01:18,148 INFO Starting Flet View app...
2025-02-06 00:01:18,148 INFO Flet View found in: C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet\bin\flet\flet.exe
2025-02-06 00:01:18,420 INFO App session started
2025-02-06 00:06:12,704 INFO Logger is configured with simple rotating file handler.
2025-02-06 00:06:12,704 INFO Assets path configured: C:\Users\K.T\Documents\Github\R_ScrapingRank\assets
2025-02-06 00:06:12,704 INFO Starting up TCP server on localhost:50806
2025-02-06 00:06:12,715 INFO Flet app has started...
2025-02-06 00:06:12,715 INFO App URL: tcp://localhost:50806
2025-02-06 00:06:12,715 INFO Starting Flet View app...
2025-02-06 00:06:12,715 INFO Flet View found in: C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet\bin\flet\flet.exe
2025-02-06 00:06:12,882 INFO App session started
2025-02-06 00:06:22,685 INFO Logger is configured with simple rotating file handler.
2025-02-06 00:06:22,686 INFO Assets path configured: C:\Users\K.T\Documents\Github\R_ScrapingRank\assets
2025-02-06 00:06:22,687 INFO Starting up TCP server on localhost:50813
2025-02-06 00:06:22,690 INFO Flet app has started...
2025-02-06 00:06:22,690 INFO App URL: tcp://localhost:50813
2025-02-06 00:06:22,690 INFO Starting Flet View app...
2025-02-06 00:06:22,691 INFO Flet View found in: C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet\bin\flet\flet.exe
2025-02-06 00:06:22,966 INFO App session started
2025-02-06 00:07:35,382 INFO Logger is configured with simple rotating file handler.
2025-02-06 00:07:35,383 INFO Assets path configured: C:\Users\K.T\Documents\Github\R_ScrapingRank\assets
2025-02-06 00:07:35,383 INFO Starting up TCP server on localhost:50825
2025-02-06 00:07:35,387 INFO Flet app has started...
2025-02-06 00:07:35,387 INFO App URL: tcp://localhost:50825
2025-02-06 00:07:35,387 INFO Starting Flet View app...
2025-02-06 00:07:35,388 INFO Flet View found in: C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet\bin\flet\flet.exe
2025-02-06 00:07:35,552 INFO App session started
2025-02-06 00:07:50,905 ERROR Future exception was never retrieved
future: <Future finished exception=TypeError("'NoneType' object is not subscriptable")>
Traceback (most recent call last):
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet_core\page.py", line 941, in wrapper
    handler(*args)
  File "c:\Users\K.T\Documents\Github\R_ScrapingRank\main.py", line 219, in on_register_new_item
    data['items'].append({
    ~~~~^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
2025-02-06 00:07:53,504 ERROR Future exception was never retrieved
future: <Future finished exception=TypeError("'NoneType' object is not subscriptable")>
Traceback (most recent call last):
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet_core\page.py", line 941, in wrapper
    handler(*args)
  File "c:\Users\K.T\Documents\Github\R_ScrapingRank\main.py", line 219, in on_register_new_item
    data['items'].append({
    ~~~~^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
2025-02-06 00:07:54,484 ERROR Future exception was never retrieved
future: <Future finished exception=TypeError("'NoneType' object is not subscriptable")>
Traceback (most recent call last):
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet_core\page.py", line 941, in wrapper
    handler(*args)
  File "c:\Users\K.T\Documents\Github\R_ScrapingRank\main.py", line 219, in on_register_new_item
    data['items'].append({
    ~~~~^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
2025-02-06 00:08:03,846 ERROR Future exception was never retrieved
future: <Future finished exception=TypeError("'NoneType' object is not subscriptable")>
Traceback (most recent call last):
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet_core\page.py", line 941, in wrapper
    handler(*args)
  File "c:\Users\K.T\Documents\Github\R_ScrapingRank\main.py", line 219, in on_register_new_item
    data['items'].append({
    ~~~~^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
2025-02-06 00:11:13,541 INFO Logger is configured with simple rotating file handler.
2025-02-06 00:11:13,541 INFO Assets path configured: C:\Users\K.T\Documents\Github\R_ScrapingRank\assets
2025-02-06 00:11:13,541 INFO Starting up TCP server on localhost:50845
2025-02-06 00:11:13,541 INFO Flet app has started...
2025-02-06 00:11:13,541 INFO App URL: tcp://localhost:50845
2025-02-06 00:11:13,541 INFO Starting Flet View app...
2025-02-06 00:11:13,541 INFO Flet View found in: C:\Users\K.T\AppData\Local\Programs\Python\Python312\Lib\site-packages\flet\bin\flet\flet.exe
2025-02-06 00:11:13,713 INFO App session started
